ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2690,6 +2690,29 @@ periodics:
       resources:
         requests:
           cpu: 500m
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@daily'
+  decorate: true
+  name: periodic-openshift-release-qci-pruner
+  spec:
+    containers:
+    - args:
+      - --confirm=false
+      command:
+      - ./qci_registry_pruner.py
+      env:
+      - name: QUAY_OAUTH_TOKEN_ENV_NAME
```

Comment:
```suggestion
      - name: QUAY_OAUTH_TOKEN
```

Reply:
Should we keep it the same as https://github.com/openshift/release/pull/53182/files#diff-5d7c76ce5aadd38b6f0b95f1f529c461c621319613ecc02ecd3c9aefd73181afR92 ?